assets/migrations/postgres/003_reverse_lookup_index.sql

```diff
@@ -0,0 +1,5 @@
+-- +goose Up
+CREATE INDEX idx_reverse_lookup_user on tuple (store, object_type, relation, _user);
```

Comment:
This index is needed to ensure the `ReadStartingWithUser` storage method produces index scans on larger collections of relationship tuples instead of sequence scans.